2d/absorbing_example/mpm.json

```diff
@@ -0,0 +1,90 @@
+{
+  "title": "Absorbing-Boundary",
+  "mesh": {
+    "mesh": "mesh.txt",
+    "entity_sets": "entity_sets.json",
+    "cell_type": "ED2Q4",
+    "isoparametric": false,
+    "io_type": "Ascii2D",
+    "node_type": "N2D",
+    "boundary_conditions": {
+      "velocity_constraints": [
+        { "nset_id" : 0, "dir" : 0, "velocity" : 0.0 }
+      ],
+      "absorbing_constraints" : [
+        { 
+          "nset_id" : 1,
+          "dir" : 1,
+          "delta" : 0.10,
+          "h_min" : 0.001,
+          "a" : 1.0,
+          "b" : 1.0,
+          "position" : "corner"
+        },
+        { 
+          "nset_id" : 2,
+          "dir" : 1,
+          "delta" : 0.10,
+          "h_min" : 0.001,
+          "a" : 1.0,
+          "b" : 1.0,
+          "position" : "bottom"
```

Comment:
Naming convention has been updated to handle 3D. This needs to be updated prior to merge